Machine Learning/sklearn examples/K Mean Clustering/K-Mean Clustering .ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<p>Not something really important but just to maintain consistency could you unbold the explanation text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
That dataset name ?

Reply:
Oh, I now understand this discussion isn't clearly visible outside ReviewNB, I meant this piece of text: 

> Here we used only two coluns petal length and petal width as they give most of the information compared to other columns and thus makes the clustering more effectively.